osu.Game.Tests/Visual/Online/TestSceneHistoricalSection.cs

```diff
@@ -37,8 +38,8 @@ public TestSceneHistoricalSection()
                 Child = section = new HistoricalSection(),
             });
 
-            AddStep("Show peppy", () => section.User.Value = new APIUser { Id = 2 });
-            AddStep("Show WubWoofWolf", () => section.User.Value = new APIUser { Id = 39828 });
+            AddStep("Show peppy", () => section.UserProfileData.Value = new UserProfileData(new APIUser { Id = 2 }));
```

Comment:
Would you be against leaving this as `.User`? I don't feel like it's misleading to do so, and will reduce the scope of changes.

Reply:
That is not easily done because of how this particular property is inherited via `ProfileSection`. Some will need the full profile data, some won't, necessarily. https://github.com/ppy/osu/pull/22006 touches on this somewhat.

Reply:
To me, I find it better shortening the property name, `UserProfileData` as a property name feels a bit too verbose.

Reply:
I fail to see how that helps reduce the scope of these changes if the line still has to change because the type changes. But if you insist I'll go back and change the naming a third time.

Reply:
*throws up hands*

Fine, applied in https://github.com/ppy/osu/pull/22005/commits/62e12277d826793da660871d0b8d44dc9e847356. I think.

I hope you don't decide you hate the `User.Value.User` and `user.NewValue.User` sort of wonkage that this spawns, because if you do and I have to revert course again, I think I'll have to check myself into a mental facility.

Reply:
Fortunately I'm okay with those kinds of cases.